<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备工具箱</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icon/Icon/DeviceToolbox.ico</normaloff>:/Icon/Icon/DeviceToolbox.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="9" colspan="2">
    <widget class="QPushButton" name="aboutButton">
     <property name="text">
      <string>关于(&amp;A)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="connectButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>连接设备</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="refreshButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>清空列表</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="7">
    <widget class="QToolButton" name="upgrade">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>开始升级</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="findAndStopButton">
     <property name="text">
      <string>搜索设备</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="configButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>配置设备</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="restartButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>重启</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="11">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>130</number>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>设备IP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备型号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MAC地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>连接状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>升级信息</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="8">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>findAndStopButton</tabstop>
  <tabstop>refreshButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>configButton</tabstop>
  <tabstop>restartButton</tabstop>
  <tabstop>treeWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
